filetracker/client/remote_data_store.py

```diff
@@ -171,8 +172,8 @@ def delete_file(self, filename):
         url, version = self._parse_name(filename)
         response = requests.delete(url
                                    + "?" + self._encode_url_params(version))
-        # SIO-2093
-        # response.raise_for_status()
+        # SIO-2093 (TODO check if nothing breaks in oioioi)
```

Comment:
TODO

Reply:
I thought that we'd check when integrating with oioioi after this pr is merged. By I guess I could test that now...

Reply:
If possible, it'd be better to test now than having a TODO for later.

Reply:
Soo, this in particular seems not to break anything - deleting works correctly and everything is alright.
Buuut... while testing that I discovered that uploading a problem package doesn't work. I don't think this particular change is what broke it, so I'm not sure if we should suspend it because of that.

I'm still investigating what's going in with the packages.
I get this error:
`Version /package/progr/test_simple_package_dXcfCTl.zip@1526156476 not available. Server has 1526149283` and I'm looking for its cause. The strange thig is that we **theoretically** didn't change any versioning semantics.

Reply:
I have tested deleting and it works correctly if other issues are resolved.
The other issues are however not in the scope of this PR (they've come about with the first PR introducing new filetracker storage architecture), so I suggest this PR to be merged, allowing us to fix these issues in another one.